src/index.js

```diff
@@ -52,3 +56,7 @@ export const shutdown = guard('shutdown');
 export const restart = guard('restart');
 export const anonymize = guard('anonymize');
 export const init = initOnce(_init, 'FullStory init has already been called once, additional invocations are ignored');
+export const IFrameOption = {
```

Comment:
I don't think you need this. The enum is defined in the typedef which JavaScript devs can reference to know what the expected values are.

```
var options = {
  // ... other stuff
  iFrameOption: '_fs_run_in_iframe'
}
```

Reply:
Removal of the export causes issues for the consuming Angular/React ref apps.  Specifically, it throws `"export 'IFrameOption' (imported as 'FullStory') was not found in '@fullstory/browser'` and `Attempted import error: 'IFrameOption' is not exported from '@fullstory/browser' (imported as 'FullStory').` during builds.  My assumption is that because were declaring something that does not exist in the actual implementation.